quickwit-metastore/src/metastore_resolver.rs

```diff
@@ -100,11 +101,28 @@ pub fn quickwit_metastore_uri_resolver() -> &'static MetastoreUriResolver {
                 )
         }
 
+        #[cfg(feature = "azure")]
+        {
+            builder = builder.register("azure", FileBackedMetastoreFactory::default());
+        }
+
+        #[cfg(not(feature = "azure"))]
+        {
+            builder = builder.register(
+                "azure",
+                UnsuportedMetastore {
+                    message: "azure unsupported, quickwit was compiled without the `azure` \
+                              feature flag"
+                        .to_string(),
+                },
+            )
+        }
+
         builder.build()
     })
 }
 
-/// A postgres metastore factory
+/// A metastore factory for handling not supported features.
```

Comment:
```suggestion
/// A metastore factory for handling unsupported metastore.
```

I guess it's the case for any unsupported protocol, behind a feature flag or not.

Reply:
yes